modules/reference/pages/configuration/config-channels.adoc

```diff
@@ -1,3 +1,134 @@
+[[ref.webui.config.channels]]
 = Configuration Channels
 
+As mentioned above, {productname} manages both central and local configuration channels and files.
+Central configuration management allows you to deploy configuration files to multiple systems (both traditional clients ([Management]) and Salt minions ([Salt])).
+Local configuration management is available for traditional clients ([Management]) only and allows you to specify overrides or configuration files that are not changed by subscribing the system to a central channel.
+
+A "`state channel`" is a type of a configuration channel but for Salt minion only.
+For a state channel, the [path]``init.sls`` file is not auto-generated, the user creates and edits it.
+Additionally, state channels can contain arbitrary configuration files that could be referenced from within the [path]``init.sls`` file.
+Therefore, state channels effectively replace custom states.
+
+.Referencing Configuration Files with Organization ID
+[NOTE]
```

Comment:
Move this admonition to the step it directly relates to, rather than at the top of the procedure.